src/extras.jl

```diff
@@ -41,9 +41,17 @@ function fill_refs!(refs::AbstractArray, X::AbstractArray{>: Missing},
     end
 end
 
+"""
+    _default_formatter_
+    
+Provide the default label format for the `cut` function.
+"""
+default_formatter(from, to, i; extend=false) = string("[", from, ", ", to, extend ? "]" : ")")
```

Comment:
Actually I realize `extend` isn't the best term here, since it's quite different from the argument passed to `cut`. Maybe better call it `closed`, i.e. it indicates whether the interval is closed on both sides or not.